src/pymor/reductors/basic.py

```diff
@@ -39,10 +39,18 @@ class ProjectionBasedReductor(BasicObject):
     check_tol
         If `check_orthonormality` is `True`, the numerical tolerance with which the checks
         are performed.
+    assemble_error_estimate
+        If `True`, assembles error estimate
+    assemble_output_error_estimate
+        If `True`, assembles output error estimate (implies `assemble_error_estimate=True`).
     """
```

Comment:
I gave this some more thought, and I think we should not add this feature: Right now, whether or not to have an error estimator is handled via inheritance. (This is the only difference between `CoerciveRBReductor` and `StationaryRBReductor`.) I think this is bad design and should be solved in a different way in the future. But for now, if I don't want to assemble the estimator, I should use the reductor without the estimator. 

Further, I do not see need for being able to disable assembly of the output estimator. Assembling the output_estimator should be generally much cheaper than assembling the state space estimator, so I think it is fine to always assemble it together with the state space estimator.

Reply:
Ah yes that's actually true, I have not thought about that, yet. Then we should definitely remove the `assemble_error_estimate` from `ProjectionBasedReductor` and the like.

Regarding the `assemble_output_error_estimate`, I only party agree. There probably exist cases where the extra work is at least countable. However, for keeping things simple, we can of course just remove the optional assembly and pay the extra work in any case. The idea of the optional assembly anyway initially came from the fact that the DWR output should not always be constructed. However, we outsourced the DWR approach which somehow makes `assemble_output_error_estimate` obsolete.